### PR TITLE
Story/115 passdown specification items

### PR DESCRIPTION
**Pull request by redcatbear**

### CODE REVIEW

File: doc/system_requirements.md [hunk 1]
```diff
@@ -183,6 +183,21 @@ Tags are a way to label an artifact intended for grouping.
...
 Needs: dsn
 
+#### Forwarding Needed Coverage
+`req~forwarding_needed_coverage~1`
+
+OFT allows users to forward needed coverage from one artifact type to one or more different types.
+
+Rationale:
+
+Often a system requirement does not affect the highlevel architecture in this case it is convenient for the architect to hand it down to lower levels directly.
```
Comment (kaklakariada) [inline doc/system_requirements.md]: Start a new sentence at "in this case "

File: doc/system_requirements.md [hunk 2]
```diff
@@ -238,6 +255,21 @@ Covers:
 
 Needs: dsn
 
+##### Artifact Type Forwarding in Markdown
+`req~artifact-type-forwarding-in-markdown~1`
+
+The Markdown format features a compact way to forward a requirement from the artifact type that war originally required to one or more different artifact types.
```
Comment (kaklakariada) [inline doc/system_requirements.md]: war -> was

File: doc/system_requirements.md [hunk 3]
```diff
@@ -238,6 +255,21 @@ Covers:
...
+
+Rationale:
+
+Often a system requirement does not affect the highlevel architecture in this case it is convenient for the architect to hand it down to lower levels directly.
```
Comment (kaklakariada) [inline doc/system_requirements.md]: Start a new sentence at "in this case"

File: oft [hunk 4]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+script_path=$(dirname "$(readlink -f "$0")")
+version=$(grep -zo '<version>.*</ver.*\n.*<name>OpenFastTrace' pom.xml | sed -e's/[^0-9.]//g')
+
+/usr/bin/java -jar "$script_path/target/openfasttrace-$version.jar" $@
```
Comment (kaklakariada) [inline oft]: Don't hard code the java path. Better rely on $PATH

File: doc/system_requirements.md (hunk 1 above)
Comment (redcatbear) [reply doc/system_requirements.md]: Fixed.

File: .circleci/config.yml [hunk 5]
```diff
@@ -32,7 +32,3 @@ jobs:
         
       # run tests!
       - run: mvn integration-test
-
-      - store_test_results:
-          path: target/surefire-reports
```
Comment (kaklakariada) [inline .circleci/config.yml]: Why do you remove this? This way circleci won't read and display test results.

Comment (redcatbear) [reply .circleci/config.yml]: Merging error. Will reintroduce it.

**Pull request merged**